src/elixirSenseAutocompleteProvider.ts

```diff
@@ -76,7 +72,7 @@ export class ElixirSenseAutocompleteProvider implements vscode.CompletionItemPro
         const matchesWordEnd = prefix.match(/\.[^A-Z][^\.]*$/);
         const matchesNonWordEnd = prefix.match(/^[^A-Z:][^\.]*$/);
         const isPrefixFunctionCall = !!(matchesWordEnd || matchesNonWordEnd);
-        if (prefix !== '' && isPrefixFunctionCall) {
+        if (prefix && !isPrefixFunctionCall) {
```

Comment:
@fr1zle fixed the module dot autocomplete. I was missing a `!`
compare to the original: https://github.com/fr1zle/vscode-elixir/pull/75/files#diff-7591485bae15ac02d0a3b1808e688fa3L68